js/scenes/credits.js

```diff
@@ -1,7 +1,6 @@
 define(['lib/scene', 'entity/back', 'basic/text', 'geo/v2'],
 		function(Scene, BackButton, TextEntity, V2) {
 			function CreditsScene() {
-				Scene.call(this);
```

Comment:
Please leave the constructor call the to be consistent with the rest of the project also it might not be needed in this place